index.html

```diff
@@ -1 +1,9 @@
-Emmely Suarez
+
+<center>
```

Comment:
`<center>` is now deprecated, and you'll want to include `<html><body>...`

Reply:
Got it. Does that mean I have to create a new branch with the updated changes?